### PR TITLE
[#5778] feat(aliyun-bundles)support OSS secret key credential

### PR DESCRIPTION
**Pull request by sunxiaojian**
### What changes were proposed in this pull request?

Support OSS secret key credential

### Why are the changes needed?

Fix: [# (5778)](https://github.com/apache/gravitino/issues/5778)

### How was this patch tested?

IcebergRESTOSSSecretIT

### CODE REVIEW

**Comment by sunxiaojian** [comment]
@FANNG1 PTAL

File: api/src/main/java/org/apache/gravitino/credential/OSSSecretKeyCredential.java [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+  /**
+   * Constructs an instance of {@link OSSSecretKeyCredential} with the static OSS access key ID and
+   * secret access key.
+   *
+   * @param accessKeyId The OSS static access key ID.
+   * @param secretAccessKey The OSS static secret access key.
+   */
+  public OSSSecretKeyCredential(String accessKeyId, String secretAccessKey) {
+    Preconditions.checkNotNull(accessKeyId, "OSS access key Id should not null");
```
Comment (FANNG1) [inline api/src/main/java/org/apache/gravitino/credential/OSSSecretKeyCredential.java:48]: could you reuse `validate` here?

**Comment by FANNG1** [comment]
LGTM except for minor comments, 
1.  add test in `TestCredentialFactory`
2. remove `ICEBERG_OSS_ACCESS_KEY_ID` and `ICEBERG_OSS_ACCESS_KEY_SECRET` from `catalogPropertiesToClientKeys` in `CatalogWrapper` as this OSS secret key provider is supported in this PR.

**Comment by sunxiaojian** [comment]
> LGTM except for minor comments, could you do extra work?
> 
> 1. add test in `TestCredentialFactory`
> 2. remove `ICEBERG_OSS_ACCESS_KEY_ID` and `ICEBERG_OSS_ACCESS_KEY_SECRET` from `catalogPropertiesToClientKeys` in `CatalogWrapper` as this OSS secret key provider is supported in this PR.

ok， fixed

File: api/src/main/java/org/apache/gravitino/credential/OSSSecretKeyCredential.java (hunk 1 above)
Comment (sunxiaojian) [reply api/src/main/java/org/apache/gravitino/credential/OSSSecretKeyCredential.java]: fixed

**Review by FANNG1 — APPROVED** [review]

**Comment by FANNG1** [comment]
LGTM

**Pull request merged**